client/src/pages/ProfileDetails/BookingForm/useStyles.ts

```diff
@@ -0,0 +1,80 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  requestForm: {
+    boxShadow: 'rgba(0, 0, 0, 0.24) 0px 3px 8px',
+    width: '80%',
+    minWidth: 400,
+    height: '55%',
+    borderRadius: 5,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifySelf: 'flex-end',
+  },
+  userRate: {
+    marginTop: 40,
+    color: 'black',
+    fontSize: 22,
+    fontWeight: 'bold',
+  },
+  userRatingContainer: {
+    marginTop: 10,
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  userRateStar: {
+    height: 18,
+    width: 'auto',
+  },
+  profileDetailsForm: {
+    width: '100%',
+    marginTop: 50,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  profileDetailsLabel: {
+    color: 'black',
+    fontSize: 14,
+    fontWeight: 'bold',
+    textTransform: 'uppercase',
+    alignSelf: 'flex-start',
+    marginLeft: '10%',
+  },
+  profileDetailsFormContainer: {
+    width: '80%',
+    display: 'grid',
+    gridTemplateColumns: '50fr 50fr',
+  },
+  profileDetailsFormInput: {
+    borderTopRightRadius: 0,
+    borderBottomRightRadius: 0,
+  },
+  profileDetailsFormSelect: {
+    borderTopLeftRadius: 0,
+    borderBottomLeftRadius: 0,
+  },
+  requestFormButton: {
+    marginTop: 70,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    padding: 10,
+    width: 160,
+    height: 56,
+    borderRadius: theme.shape.borderRadius,
+    marginBottom: 35,
+    fontSize: 14,
+    backgroundColor: theme.palette.primary.main,
+    color: 'white',
+    fontWeight: 'bold',
+    textTransform: 'uppercase',
+    transition: '1s',
+    '&:hover': {
+      backgroundColor: '#30E664',
+    },
+  },
```

Comment:
This file in general, especially this component, contains too much css for it to be "right". Main benefit of material ui is that it facilitates achieving consintent app-wide styling, and using arbitrary css classes like these strongly goes against that. The ideal approach would be to apply as much styling as possible through the available props of the material ui components (for example, see the docs for a [complete list of props for `Button`](https://v4.mui.com/api/button/#props), and [some usage examples](https://v4.mui.com/components/buttons/)).

In this particular case, we can probably cut the styles listed here in half simply by using `color` and `size` props. You can always [customize the theme](https://v4.mui.com/customization/theming/#theming) for it to suit your styling/mocks but keep in mind that those changes will affect the entire application. The tricky part of material ui is to identify what should be a app-wide style and what should be a one-off customization properly, but the rest is made very simple so it's very often worth the effort 👍🏼

Reply:
I didn't want to customize the theme as there are many other components that I did not work on that would be affected by my changes, and it would be too time-consuming to try to troubleshoot all the issues this would generate. I decided to apply styles directly to unique elements via props. I kept the class within the useStyles file for the classes used by multiple elements.